ssr-opengraph/src/index.tsx

```diff
@@ -6,11 +6,27 @@ import { collectionDetail, galleryDetail, userDetail } from './handlers';
 
 const app = new Hono();
 
-app.get('/', (c) => {
-  return c.text('hello hono.js');
-});
+const chains = ['bsx', 'snek', 'rmrk', 'ksm', 'stmn'];
+
+app.get('/', async(c) => {
+  const useragent = c.req.headers.get('user-agent');
+
+  if (useragent && !isbot(useragent)) {
+    return fetch(c.req.url);
+  }
+
+  const props = {
+    name: 'KodaDot',
+    siteData: {
+      title: 'KodaDot - NFT Market Explorer',
+      description: 'One Stop NFT Shop on Polkadot',
+      canonical: 'https://kodadot.xyz',
+      image: 'https://kodadot.xyz/k_card.png',
+    }
```

Comment:
So if I understand the one set on `nuxt.config.js` cannot be taken into account?

Reply:
correct, what we change in our `nuxt.config.js` won't reflect in our prerendering case for now